CONTRIBUTING.md

```diff
@@ -1,45 +1,62 @@
-Where do I start?
-=================
-
-See issues tagged with [Pull Request
-Welcomed](https://github.com/talha131/pelican-elegant/issues?labels=Pull+Request+Welcomed).
-I, @talha131, do not plan to work on these issues. You are more than welcome to
-pick them up.
-
-New Features and Styles
-=======================
-
-If you plan to add new features to the theme, please make sure
-
-1. You set sensible defaults so that theme works out of the box, without
-   forcing user to set any variable
-2. Your feature should not effect readability and reading experience
-3. It should not be distracting for the reader
-
-Follow the code style of the existing codebase
-==============================================
-
-1. Use single ('') rather than double ("") quotation marks for Jinja strings
-1. In Jinja print statements, surround the variable with spaces inside curly
-   braces, for example `{{ foo.bar }}`
-1. Use double ("") quotation marks around HTML attributes
-1. End files with a newline
-
-CSS Formatting Rules
---------------------
-
-1. Font name's first letter should be capital
-2. Add a space after comma
-3. Declarations should be sorted alphabetically
-4. Use a single space between the last selector and the opening brace that
-   begins the declaration block
-5. Group together related classes and identities
-6. Add a space after colon
-7. Remove leading 0s
-8. Remove unit specification after 0 values
-9. Use three digit Hex notation for colors whereever possible
-10. User hyphen `-` instead of underscore `_` in class and identity names
-
-Refer to [Google's HTML/CSS Style
-Guide](http://google-styleguide.googlecode.com/svn/trunk/htmlcssguide.xml) for
-all other formatting rules.
+# Where do I start?
+
+Developing Elegant is a community effort, so you are very welcome to help develop it further into the most elegant theme out there.
+
+The main repository of Elegant is on [GitHub][elegant], which you may fork and then submit pull requests to, in order for them to be merged.
+
+If you found any issues, or have ideas how to improve the theme, please submit an [issue][].
+
+Also see issues tagged as [Pull Request Welcome](https://github.com/Pelican-Elegant/pelican-elegant/labels/pull%20request%20welcome) – these are issues that are not directly on our roadmap, but if you have time to contribute, we would be very glad to review and accept your pull request.
```

Comment:
these are `the` issues that are not on our roadmap..

Reply:
It should work both with or without `the`, but I think it’s slightly better without, as you usually use “the” when you have already identified the specific thing(s).

If we take a living thing as an example, it’s perhaps easier to visualise. “These are brown cats” means that these are cats that are brown. While “These are the brown cats” would mean that these are some brown cats that we already spoke about and we know already which specific cats we are talking about. Or if we put it in singular it would be similar to saying “this is an issue that is not directly on our roadmap” vs “this is the issue that is not directly on our roadmap”.

Reply:
No issues. English is not my first language so I am going to trust your judgement on this. Thanks for the explanation.

Reply:
It’s not mine either, but I’m pretty sure about this. If a native speaker has a better suggestion, I agree we should fix it :)